ui/templates/includes/resource_panel.html

```diff
@@ -11,6 +11,11 @@
             <span class="hidden-xs">XML</span>
           </a>
         </li>
+        <li>
```

Comment:
It's branch pollution; it's in its own branch and I'll delete it from here and squash.